tests/main/snap-quota-groups/task.yaml

```diff
@@ -0,0 +1,107 @@
+summary: Functional test for quota-related snap commands.
+
+details: |
+  Functional test for snap quota group commands ensuring that they are 
+  effective in practice.
+
+prepare: |
+  if os.query is-trusty; then
+    exit 0
+  fi
```

Comment:
Can you use` systems: ...` instead?

Reply:
We could but then we would have to have two spread tests, one for trusty and one for !trusty. No strong preference either way

Reply:
Ah, I forgot the test also checks that we detect old systemd below, so scratch my comment.